pandas/core/algorithms.py

```diff
@@ -267,22 +267,84 @@ def match(to_match, values, na_sentinel=-1):
     return result
 
 
-def unique1d(values):
+def unique(values):
     """
-    Hash table-based unique
+    Hash table-based unique. uniques are returned in order
+    of appearance. This does NOT sort.
+
+    Parameters
+    ----------
+    values : 1d array-like
+
+    Returns
+    -------
+    unique values.
+    - If the input is a Categorical dtype, the return is a Categorical
```

Comment:
Those bullet points need one level of indentation (they are the 'description' part like in the parameters section)